modules/aerodyn/src/AirfoilInfo.f90

```diff
@@ -683,7 +657,7 @@ SUBROUTINE ReadAFfile ( AFfile, NumCoefs, InCol_Alfa, InCol_Cl, InCol_Cd, InCol_
                CALL ParseVarWDefault ( FileInfo, CurLine, 'filtCutOff', p%Table(iTable)%UA_BL%filtCutOff, 0.5_ReKi, ErrStat2, ErrMsg2, UnEc )
                   CALL SetErrStat( ErrStat2, ErrMsg2, ErrStat, ErrMsg, RoutineName )
                   
-               p%ColUAf    = NumCoefsTab + 1 ! column for f_st
+               p%ColUAf    = NumCoefsIn + 1 ! column for f_st
```

Comment:
Is this a regression?  We ran into issues with multiple tables and corrected it with PR #627, but this PR seems to be changing it back to the problematic code.

Reply:
@andrew-platt , this is code from my recent UA pull request #647. I don’t think this is actually changing it back to problematic code. It is just modifying it in a different way. Note that the number of coefficients passed as an argument is no longer intent(out).

Reply:
Thanks for the clarification. I'll look at the logic more closely.

Do you know if this code can handle one of the airfoil files containing more tables than the other files?  What I had found in #627 was that the numcoefs wouldn't get reset between the reading of the tables and the array would get overstepped (perhaps my solution actually introduced other issues).

Reply:
Yes. In the AD input file, you must specify which columns are in (all) the airfoil tables. Each table must include at least those numbers of columns. Those inputs determine the number of columns being read from the file... you could theoretically have more columns in the tables, but they wouldn't be used. When adding the new HGM unsteady aero model, we need to calculate 2 more columns to be interpolated with alpha, but I messed up the logic if there were multiple tables in each file. So now, the number of columns as determined from the AD file (`NumCoefsIn`) is fixed, but then each table in each airfoil file has the number of columns calculated independently (`NumCoefsTab` is either (1) the number from AD or (2) the number from AD + 2). The 3D spline coefficients for each table are now allocated based on the size of the coefficients array instead of the `NumCoefs` variable, so you don't have to do the check that the arrays are the same size later. 

With this method, you do not have to specify UA coefficients for each table unless you are actually using a UA model. (e.g., you could specify UA coefficients for the first table but not the second and it would still run for cases where UA was not used.) I didn't want to have AFI checking UA stuff any more than it needs to, and it's a little clearer to me what that NumCoefs in the calling routine is actually doing if we don't modify it in this subroutine.